### ISSUE TITLE
[Question]: FutureWarning on python-3.10

### ISSUE THREAD

**Issue by shigechika** [J-Quants/jquants-api-client-python#48]
### Question form pre-submit checklist.

- [X] I have searched the existing issues to ensure there isn't already an issue about this question.
- [X] My question has to do with the Python SDK and isn't a general question about the API. (If it is please open your issue in J-Quants API Community on Slack)
- [X] My question isn't about how to do a specific algorithm or asking for trade advice (answers to these are outside the scope of this repo).

### Question

I am using jquantsapi on python-3.10.

```
cli = jquantsapi.Client()
df = cli.get_price_range( ... )
```

get_price_range() dump warnings. 
Could you quiet the following messages?

```
/usr/local/lib/python3.10/site-packages/jquantsapi/client.py:537: FutureWarning: In a future version, `df.iloc[:, i] = newvals` will attempt to set the values inplace instead of always setting a new array. To retain the old behavior, use either `df[df.columns[i]] = newvals` or, if columns are non-unique, `df.isetitem(i, newvals)`
  df.loc[:, "Date"] = pd.to_datetime(df["Date"], format="%Y%m%d")
```

**Comment by hiohiohio** [comment]
@shigechika Thank you for the report. Sure. Please let me fix this for the next release hopefully. cc: @s-meitoma

**Comment by hiohiohio** [comment]
@shigechika We have released fixed version, [v0.2.2](https://github.com/J-Quants/jquants-api-client-python/releases/tag/v0.2.2). Can you please update the client? Thank you!

**Comment by shigechika** [comment]
Thank you. I checked  get_price_range() became quiet😀
Please close this case.
